Source/CesiumRuntime/Private/CesiumGeoreferenceComponent.cpp

```diff
@@ -236,17 +259,17 @@ void UCesiumGeoreferenceComponent::_updateFromActor() {
 }
 
 glm::dvec3 UCesiumGeoreferenceComponent::_getAbsoluteLocationFromActor() {
-  const UWorld* world = this->GetWorld();
-  if (!IsValid(world)) {
+  const UWorld* pWorld = this->GetWorld();
```

Comment:
Mixing cleanups like this with the actual fix makes it difficult to sort that out later...

Reply:
Yes, sorry, I meant to undo that before opening this PR.